include/dem/insertion.h

```diff
@@ -40,7 +40,7 @@ using namespace dealii;
  * @brief Base interface for classes that carry out the insertion of particles
  * in the system.
  */
```

Comment:
Add template parameter to documentation of the class, for example following the same idea as the `ParticleProperties `class in `dem_properties.h`:
```suggestion
 * @brief Base interface for classes that carry out the insertion of particles
 * in the system.
 * @tparam dim An integer that denotes the number of spatial dimensions.
 * @tparam PropertiesIndex Index of the properties used within the ParticleHandler.
 */
```